docs/_providers/netcup.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Netcup
+title: Netcup Provider
+layout: default
+jsId: Netcup
+---
+# Netcup Provider
+
+## Configuration
+In your credentials file, you must provide your [api key, password and your customer number](https://www.netcup-wiki.de/wiki/CCP_API#Authentifizierung).
+
+{% highlight json %}
+{
+  "netcup": {
+    "api-key": "abc12345",
+    "api-password": "abc12345",
+    "customer-number": "123456"
+  }
+}
+{% endhighlight %}
+
+## Usage
+Example Javascript:
+
+{% highlight js %}
+var REG_NONE = NewRegistrar('none', 'NONE')
+var NETCUP = NewDnsProvider("netcup", "NETCUP");
```

Comment:
Please use `'` instead of `"` in JavaScript. (I know some of the docs have the wrong one, but I'll be fixing that soon.)

(FYI: don't change the quotes in the JSON example... that has to be `"`)

Reply:
ah, sure :)